gadget/params.c

```diff
@@ -300,7 +300,7 @@ create_gadget_parameter_set()
     param_declare_int(ps, "MassiveNuLinRespOn", REQUIRED, 0, "Enables linear response massive neutrinos of 1209.0461. Make sure you enable radiation too.");
     param_declare_int(ps, "HybridNeutrinosOn", OPTIONAL, 0, "Enables hybrid massive neutrinos, where some density is followed analytically, and some with particles. Requires MassivenuLinRespOn");
     param_declare_string(ps, "LinearTransferFunction", OPTIONAL, "camb_transfer_99.dat", "File containing linear transfer function in CAMB format. Used for massive neutrinos.");
-    param_declare_double(ps, "InputSpectrum_UnitLength_in_cm", OPTIONAL, 3.085678e24, "Units of the CAMB transfer function in cm. By default Mpc.");
+    param_declare_double(ps, "InputSpectrum_UnitLength_in_cm", OPTIONAL, CM_PER_MPC, "Units of the CAMB transfer function in cm. By default Mpc.");
```

Comment:
Mpc/h or Mpc?

Reply:
Units of the CAMB transfer function in cm/h, by default Mpc/h?

Reply:
Fixed, thanks.